pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -49,6 +49,8 @@ class Hadamard(Observable, Operation):
     num_params = 0
     """int: Number of trainable parameters that the operator depends on."""
 
+    _queue_category = "_ops"
+
```

Comment:
What about all of the other gates in this file ? What about templates? Will this be added to those afterwards ?